src/physics/initial_conditions/initial_condition.cpp

```diff
@@ -169,15 +172,23 @@ InitialConditionFactory<dim,real>::create_InitialConditionFunction(
 // ========================================================
 // ZERO INITIAL CONDITION
 // ========================================================
-template<int dim, typename real>
-real InitialConditionFunction_Zero<dim, real> :: value(const dealii::Point<dim,real> &/*point*/, const unsigned int /*istate*/) const 
+template <int dim, int nstate, typename real>
+real InitialConditionFunction_Zero<dim, nstate, real> :: value(const dealii::Point<dim,real> &/*point*/, const unsigned int /*istate*/) const
```

Comment:
Just wondering, how come `InitialConditionFunction_Zero` is not in the `create_InitialConditionFunction()` ?

Reply:
Ok I think it's fine then